build/build.sh

```diff
@@ -16,6 +16,13 @@ if [[ $OSDIST == "centos" ]] || [[ $OSDIST == "amzn" ]] || [[ $OSDIST == "rhel"
     fi
 fi
 
+CMAKE_EXTRA_DEF=""
+
+if [[ $OSDIST == "ubuntu" ]]; then
+    OS_VERSION_ID=`grep '^VERSION_ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
```

Comment:
OS_VERSION_ID is extracted but not used?

Reply:
Otherwise for this we just need to `source /etc/os-release`